src/duplicacy_snapshotmanager.go

```diff
@@ -1585,15 +1616,24 @@ func (manager *SnapshotManager) PruneSnapshots(selfID string, snapshotID string,
 		LOG_WARN("DELETE_OPTIONS", "Tags or retention policy will be ignored if at least one revision is specified")
 	}
 
-	preferencePath := GetDuplicacyPreferencePath()
-	logDir := path.Join(preferencePath, "logs")
-	os.Mkdir(logDir, 0700)
+	prefPath := GetDuplicacyPreferencePath()
+	logDir := path.Join(prefPath, "logs")
+	err := os.MkdirAll(logDir, 0700)
+	if err != nil {
+		LOG_ERROR("LOG_DIR", "Could not open log directory %s: %v", logDir, err)
+	}
 	logFileName := path.Join(logDir, time.Now().Format("prune-log-20060102-150405"))
 	logFile, err := os.OpenFile(logFileName, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
+	if err != nil {
+		LOG_ERROR("LOG_FILE", "Could not open log file %s: %v", logFileName, err)
+	}
 
 	defer func() {
 		if logFile != nil {
-			logFile.Close()
+			cerr := logFile.Close()
+			if cerr != nil {
+				LOG_ERROR("LOG_FILE", "Could not close log file %s: %v", logFileName, cerr)
```

Comment:
This should be LOG_WARN, because this defer func can be called when a panic occurs and LOG_ERROR would cause a new panic.

Reply:
Ack